src/amber/cli/templates/app/public/dist/images/logo.svg

```diff
@@ -1 +0,0 @@
-<svg xmlns="http://www.w3.org/2000/svg" width="502" height="566"><g fill="none" fill-rule="evenodd"><path fill="#F3C095" d="M.117895 142.08393L250.562495.91603l-184.7541 305.3151-65.5352 117.9262z"/><path fill="#FFFEFC" d="M250.8729974 565.0744177L.2832957 424.5200163l65.5325685-118.0244418L250.5625196.9045315l250.5896836 140.5544042.1552393 282.084939z"/><path fill="#FBE9DB" d="M65.808395 306.23113L250.562495.91603l170.6768 203.7692z"/><path fill="#F9DEC9" d="M.273195 424.15733l-.1553-282.0734 65.6904 164.1472z"/><path fill="#FEFBF6" d="M.273195 424.15733l65.5352-117.9262 185.0646 258.8319z"/><path fill="#FFFEFC" d="M339.250895 434.67373l-273.4425-128.4426 355.4309-101.5459z"/><path fill="#FDF3EA" d="M250.872995 565.06303l-185.0646-258.8319 273.4425 128.4426z"/><path fill="#FCF0E4" d="M339.250895 434.67373l81.9884-229.9885 80.0782 219.21z"/><path fill="#FBE9DB" d="M250.872995 565.06303l88.3779-130.3893 162.0666-10.7785z"/><path fill="#FFF" d="M501.317495 423.89523l-80.0782-219.21 79.9231-62.8635z"/><path fill="#FDFCFA" d="M421.239295 204.68523L250.562495.91603l250.5999 140.9057z"/></g></svg>
```

Comment:
Is this svg dynamically generated as well?

Reply:
It's not, but there's [the same svg in the assets directory](https://github.com/amberframework/amber/blob/master/src/amber/cli/templates/app/src/assets/images/logo.svg), so I'm assuming it's meant to be managed by Webpack.